main.md

```diff
@@ -523,14 +521,16 @@ Again, the SD-JWT-R follows the same structure as the `_sd` in the SD-JWT. For E
 }
 ```
 
-The SD-JWT-R MAY contain further claims, for example, to ensure a binding
-to a concrete transaction (in the example the `nonce` and `aud` claims).
+When the holder sends SD-JWT-R to the Verifier, it MUST be a base64url-encoded 
+JWS as described in Section 5.1 of [@!RFC7515]. JWS Compact Serialization and 
```

Comment:
[5.1](https://datatracker.ietf.org/doc/html/rfc7515#section-5.1).  Message Signature or MAC Computation doesn't seem like the section that you meant to reference.